src/main/java/com/redhat/devtools/lsp4ij/LSPIJUtils.java

```diff
@@ -745,7 +749,13 @@ public static Range toRange(TextRange range, Document document) {
     public static @Nullable TextRange toTextRange(@NotNull Range range,
                                                   @NotNull Document document,
                                                   boolean adjust) {
-        return toTextRange(range, document, null, adjust);
+        return toTextRange(range, document, null, adjust, -1);
+    }
+    public static @Nullable TextRange toTextRange(@NotNull Range range,
```

Comment:
Please restore the JavaDoc comment for this original signature since it's used externally.